File: CP-Excercise/A_Yura_s_New_Name.cpp
#include <bits/stdc++.h>
using namespace std;
void solve() 
{
    string str;
    cin >>str;
    string tmp="";
    for(auto &it : str)
    {
        if (it=='_')
        {
            if (tmp.empty() || tmp.back() == '_') tmp.push_back('^');
            tmp.push_back('_');
        }
        else tmp.push_back(it);
    }

    if(tmp.back() == '_') tmp.push_back('^');
    else if(tmp.size()==1) tmp+='^';

    cout << tmp.size() - str.size() << endl;
}

int main()
{
    int test;
    cin>> test;
    while (test--)
        solve();
}

File: CP-Excercise/A_Yura_s_New_Name.py
def solve():
    s = input()
    t = ""
    for i in range(len(s)):
        if s[i] == '_':
            if len(t) == 0 or t[-1] == '_':
                t += '^'
            t += '_'
        else:
            t += s[i]
    if len(t) == 0 or t[-1] == '_':
        t += '^'
    if len(t) == 1:
        t += '^'
    print(len(t) - len(s))

t = int(input())
for i in range(t):
    solve()
